### DISCUSSION TITLE
Is it possible to get the total queued toasts from useToastState?

### DISCUSSION THREAD

**Discussion by AndrewLeedham** [adobe/react-spectrum#6556] [Q&A]
The `useToastState` hook seems to only expose `visibleToasts` with no way of checking the total length of the queue. Seems like it would be useful to show a user how many notifications they have in total. Am I missing something here?

**Comment by snowystinger** [comment]
You're not missing anything, Toasts are still in Beta and we've implemented mostly what we need in React Spectrum. However, that does seem like a useful feature. I assume you're using the hooks, so you should be able to track it if you made your own implementation of just the toast queue in the meantime. Or you could potentially use a proxy pattern in front of useToastState to track them.
https://react-spectrum.adobe.com/react-aria/useToast.html#global-toast-queue
